### PR TITLE
Add AS4242422002 peerings

### PR DESCRIPTION
**Pull request by hello-smile6**
This PR should connect my personal dn42 network's nodes to your network. My end is ready to go and will be deployed shortly.

### CODE REVIEW

File: routers/router.chi1.yml [hunk 1]
```diff
@@ -46,3 +46,12 @@
     remote_address: us-chi01.dn42.tech9.io
     remote_port: 58438
     public_key: 0kb8ffMcbx8oXZ3Ii5khOuCzmRqM2Cy0IslmrWtRGSk=
+- name: KIX-9PFS
```
Comment (jvoss) [inline routers/router.chi1.yml:49]:
```suggestion

- name: KIX-9PFS
```

Comment (jvoss) [inline routers/router.chi1.yml:49]:
```suggestion

- name: KIX-9PFS
```

File: routers/router.dal1.yml [hunk 2]
```diff
@@ -68,3 +68,12 @@
     remote_address: us-dal01.dn42.tech9.io
     remote_port: 56736
     public_key: iEZ71NPZge6wHKb6q4o2cvCopZ7PBDqn/b3FO56+Hkc=
+- name: US1-9PFS
```
Comment (jvoss) [inline routers/router.dal1.yml:71]:
```suggestion

- name: US1-9PFS
```

File: routers/router.fnc1.yml [hunk 3]
```diff
@@ -107,3 +107,12 @@
     remote_address: rt1.us.tatk.network
     remote_port: 20207
     public_key: 8EjYngEiwoUaIMjHlOTysiy2SZsQzBxEy7epNK3zMGk=
+- name: US3-9PFS
```
Comment (jvoss) [inline routers/router.fnc1.yml:110]:
```suggestion

- name: US3-9PFS
```

File: routers/router.iad1.yml [hunk 4]
```diff
@@ -33,3 +33,12 @@
     - ipv6
   wireguard:
     public_key: 2BEC3CFKpAiIK3wTStbVuHyy894p5g2JLLTajdBsUF8=
+- name: US2-9PFS
```
Comment (jvoss) [inline routers/router.iad1.yml:36]:
```suggestion

- name: US2-9PFS
```

File: routers/router.lon1.yml [hunk 5]
```diff
@@ -118,3 +118,12 @@
     remote_address: rt0.uk.tatk.network
     remote_port: 20207
     public_key: jvng3/Q+uAlmk64ol9gDHImR1GvGVMvkOmYO7USMYX0=
+- name: UK1-9PFS
```
Comment (jvoss) [inline routers/router.lon1.yml:121]:
```suggestion

- name: UK1-9PFS
```

**Review by jvoss — CHANGES_REQUESTED** [review]
Let's leave a blank line between peers to improve human readability.

File: routers/router.chi1.yml [hunk 6]
```diff
@@ -46,3 +46,14 @@
     remote_address: us-chi01.dn42.tech9.io
     remote_port: 58438
     public_key: 0kb8ffMcbx8oXZ3Ii5khOuCzmRqM2Cy0IslmrWtRGSk=
+
```
Comment (jvoss) [inline routers/router.chi1.yml:49]:
```suggestion
```

Removing an extra line.

File: routers/router.chi1.yml [hunk 7]
```diff
@@ -46,3 +46,13 @@
...
+- name: KIX-9PFS
+  asn: 4242422002
+  ipv6: fe80::2002:5
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
```
Comment (jvoss) [inline routers/router.chi1.yml:57]: Missing remote_address and remote_port.

File: routers/router.dal1.yml [hunk 8]
```diff
@@ -68,3 +68,13 @@
...
     public_key: iEZ71NPZge6wHKb6q4o2cvCopZ7PBDqn/b3FO56+Hkc=
+
+- name: US1-9PFS
+  asn: 4242422002
+  ipv6: fe80::2002:1
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
```
Comment (jvoss) [inline routers/router.dal1.yml:79]: Missing remote_address and remote_port.

File: routers/router.fnc1.yml [hunk 9]
```diff
@@ -107,3 +107,13 @@
...
     public_key: 8EjYngEiwoUaIMjHlOTysiy2SZsQzBxEy7epNK3zMGk=
+
+- name: US3-9PFS
+  asn: 4242422002
+  ipv6: fe80::2002:4
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
```
Comment (jvoss) [inline routers/router.fnc1.yml:118]: Missing remote_address and remote_port.

File: routers/router.iad1.yml [hunk 10]
```diff
@@ -33,3 +33,13 @@
...
     public_key: 2BEC3CFKpAiIK3wTStbVuHyy894p5g2JLLTajdBsUF8=
+
+- name: US2-9PFS
+  asn: 4242422002
+  ipv6: fe80::2002:2
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
```
Comment (jvoss) [inline routers/router.iad1.yml:44]: Missing remote_address and remote_port.

File: routers/router.lon1.yml [hunk 11]
```diff
@@ -118,3 +118,13 @@
...
     public_key: jvng3/Q+uAlmk64ol9gDHImR1GvGVMvkOmYO7USMYX0=
+
+- name: UK1-9PFS
+  asn: 4242422002
+  ipv6: fe80::2002:3
+  multiprotocol: true
+  extended_nexthop: true
+  sessions:
+    - ipv6
+  wireguard:
```
Comment (jvoss) [inline routers/router.lon1.yml:129]: Missing remote_address and remote_port.

**Review by jvoss — CHANGES_REQUESTED** [review]
Please add your public endpoint information. We do not currently support totally dynamic peers.

**Comment by hello-smile6** [comment]
Does every peering need a public endpoint?

**Review by jvoss — APPROVED** [review]

**Pull request merged**